examples/dynamic-listener/dynamic_listener.bal

```diff
@@ -0,0 +1,39 @@
+import ballerina/http;
+import ballerina/lang.runtime;
+
+http:Listener httpListener = new(9090);
+
+var helloService = @http:ServiceConfig {} service {
+
+    resource function sayHello(http:Caller caller,
+        http:Request req) returns error? {
```

Comment:
Can we align this with the first parameter in the previous line? Please check the other function too.

Reply:
In that case the it will become longer than 80 characters? I think we can have `returns error?` below?

Reply:
Fixed by removing `returns error?`